connect/mirror/src/main/java/org/apache/kafka/connect/mirror/DefaultConfigPropertyFilter.java

```diff
@@ -30,6 +30,9 @@ public class DefaultConfigPropertyFilter implements ConfigPropertyFilter {
     
     public static final String CONFIG_PROPERTIES_EXCLUDE_CONFIG = "config.properties.exclude";
     public static final String CONFIG_PROPERTIES_EXCLUDE_ALIAS_CONFIG = "config.properties.blacklist";
+    public static final String USE_DEFAULTS_FROM = "use.defaults.from";
+    private static final String USE_DEFAULTS_FROM_DOC = "Which cluster's defaults to use when syncing topic configurations.";
+    private static final String USE_DEFAULTS_FROM_DEFAULT = "target";
 
     private static final String CONFIG_PROPERTIES_EXCLUDE_DOC = "List of topic configuration properties and/or regexes "
                                                                 + "that should not be replicated.";
```

Comment:
Can we add a note that this will also apply to properties with default values if `use.defaults.from` is set to `source`?

Reply:
This still needs to be done, not sure why it was marked resolved?

Reply:
I had changed it `Which cluster's defaults to use when syncing topic configurations that have default values.` meaning that it only applies to properties with default values.  I have updated it to include the source part, what do you think?

Reply:
I was commenting on the `CONFIG_PROPERTIES_EXCLUDE_DOC` docstring, not  `USE_DEFAULTS_FROM_DEFAULT`. But I think this depends on the outcome of the discussion on the other thread regarding how we want to use the `ConfigPropertyFilter`.